docs/USER_RECORD.md

```diff
@@ -1181,3 +1181,9 @@ systems. It would hence look like this:
         "userName" : "grobie",
 }
 ```
+
+To query a user record from the userdb, you can use the following [varlinkctl](https://www.freedesktop.org/software/systemd/man/latest/varlinkctl.html) command:
+
+```sh
+varlinkctl call unix:///run/systemd/userdb/io.systemd.Multiplexer io.systemd.UserDatabase.GetUserRecord '{"userName": "alice", "service": "io.systemd.Multiplexer"}'
+```
```

Comment:
the `unix://` prefix is redundant, you can drop it. 

this looks good to add, but I think this would be better at the end of USER_GROUP_API.md, maybe under a new section `## Command Line Access` or so.

Reply:
Thanks, I've done the requested changes.